File: second-year/second-semester/MPP/mpp-proiect-repository-raulbaciulescu/travelAgency2/model/Dto/PurchaseDto.cs
namespace model.Dto
{
    public class PurchaseDto : Entity<long>
    {

        public long flightId { get; set; }
        public string clientName { get; set; }
        public string clientAddress { get; set; }
        public List<string> tourists { get; set; }
        public int nrOfSeats { get; set; }

        public PurchaseDto(long id, long flightId, string clientName, string clientAddress, List<string> tourists, int nrOfSeats) : base(id)
        {
            this.flightId = flightId;
            this.clientName = clientName;
            this.clientAddress = clientAddress;
            this.tourists = tourists;
            this.nrOfSeats = nrOfSeats;
        }
    }
}

File: second-year/second-semester/MPP/mpp-proiect-repository-raulbaciulescu/travelAgency/Model/src/main/java/domain/dto/PurchaseDto.java
package domain.dto;
import domain.Entity;

import java.io.Serializable;
import java.util.List;

public class PurchaseDto extends Entity<Long> implements Serializable {
    private Long flightId;
    private String clientName;
    private String clientAddress;
    private List<String> tourists;
    private int nrOfSeats;

    public PurchaseDto(Long flightId, String clientName, String clientAddress, List<String> tourists, int nrOfSeats) {
        this.flightId = flightId;
        this.clientName = clientName;
        this.clientAddress = clientAddress;
        this.tourists = tourists;
        this.nrOfSeats = nrOfSeats;
    }

    public Long getFlightId() {
        return flightId;
    }

    public String getClientName() {
        return clientName;
    }

    public String getClientAddress() {
        return clientAddress;
    }

    public List<String> getTourists() {
        return tourists;
    }

    public int getNrOfSeats() {
        return nrOfSeats;
    }
}
